cfgov/paying_for_college/disclosures/scripts/update_ipeds.py

```diff
@@ -19,7 +19,7 @@
 PFC_ROOT = Path(__file__).ancestor(3)
 # LATEST_YEAR specifies first year of academic-year data
 # So 2015 would fetch data for 2015-2016 cycle
-LATEST_YEAR = datetime.datetime.now().year - 1
+LATEST_YEAR = datetime.datetime.now().year - 2
```

Comment:
😬 This scares me a little bit. Is there some assumption about which year this code will be running?

Reply:
Scary but true.  
We initially thought we'd run the annual IPEDS update late in the year, in hopes of snagging data from the most recent academic year, but found that updates were never posted in time. In practice, we stay closer to the latest data by running the IPEDS script early in the year. That means that, in 2020, during the 2019-2020 academic year (for which there is no data), we need to specify current-year minus 2 to get the latest data, from the 2018-2019 academic year.

Reply:
Thanks for the explanation. Would it be worth adding a comment here so that in future this is easier to find?